convert2rhel/checks.py

```diff
@@ -83,6 +84,23 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_custom_repos_are_valid():
+    """To prevent failures past the PONR, make sure that the enabled custom repositories are valid.
+
+    What is meant by valid:
+    - YUM/DNF is able to find the repoids (to rule out a typo)
+    - the repository "baseurl" is accessible and contains repository metadata
+    """
+    logger.task("Prepare: Checking if --enablerepo repositories are accessible")
+    call_yum_cmd(command="clean", args="metadata")
```

Comment:
Modified Michal's log suggestion a bit:
```suggestion
    logger.task("Prepare: Checking if --enablerepo repositories are accessible")

    if not tool_opts.disable_submgr:
        logger.info("Skipping the check of repositories due to RHSM being used for the conversion.")
        return

    call_yum_cmd(command="clean", args="metadata")
```